src/index.js

```diff
@@ -81,6 +81,14 @@ class Dropzone extends React.Component {
   }
 
   onDragStart(evt) {
+    const items = getDataTransferItems(evt)
+
+    // Unless there are files being dragged,
+    // do not call the onDragStart.
+    if (!Array.isArray(items) || !items.length) {
```

Comment:
Please extract the check to a function or method and use it to make the code more DRY